x-pack/plugins/actions/server/actions_client.ts

```diff
@@ -134,25 +142,59 @@ export class ActionsClient {
       actionTypeId: result.attributes.actionTypeId as string,
       name: result.attributes.name as string,
       config: result.attributes.config as Record<string, any>,
+      isPreconfigured: false,
     };
   }
 
   /**
    * Get an action
    */
   public async get({ id }: { id: string }): Promise<ActionResult> {
+    const preconfiguredConnector = this.preconfiguredConnectors.find(
+      pConnector => pConnector.id === id
+    );
+    if (preconfiguredConnector !== undefined) {
+      return {
+        id,
+        actionTypeId: preconfiguredConnector.actionTypeId,
+        name: preconfiguredConnector.name,
+        config: preconfiguredConnector.config,
+        description: preconfiguredConnector.description,
+        isPreconfigured: true,
+      };
+    }
     const result = await this.savedObjectsClient.get<RawAction>('action', id);
 
     return {
       id,
       actionTypeId: result.attributes.actionTypeId,
       name: result.attributes.name,
       config: result.attributes.config,
+      isPreconfigured: false,
     };
   }
 
   /**
-   * Find actions
+   * Get all actions connectors with preconfigured connectors
+   */
+  public async getAll(): Promise<FindActionResult[]> {
```

Comment:
I think with the addition of this function, we should remove the `find` function and related APIs as well. Did you find any places that relied on the find API functionality (sort, search, etc)?

Reply:
I found that SIEM and Monitoring plugin use it, but it seems to be well fit with the getAll method, so I will replace it and remove find api